filestore/s3.go

```diff
@@ -149,7 +160,7 @@ func (s *S3) Head(ctx context.Context, relPath string) (*File, error) {
 	if rsp.LastModified != nil {
 		file.Modified = *rsp.LastModified
 	}
-
+	s3logger.Debugw("Successfully performed HEAD", "key", relPath)
```

Comment:
Not needed IMO.

Reply:
Without these we have no visibility into s3 success path. They are already at debug level which is disabled at runtime in favour of info level logs. I see no harm in having these considering the other detailed debug level logs.

Reply:
That's fine, especially since there is no run-time indicator if s3 writing is enabled or not. Generally, I avoid logging in the success path as long as non-success paths have logging or error returns, so this was those reflexes.